dd-sdk-android/src/test/kotlin/com/datadog/android/log/LoggerTest.kt

```diff
@@ -52,16 +52,16 @@ internal class LoggerTest {
 
     @BeforeEach
     fun `set up`(forge: Forge) {
-        LogsFeature.envName = ""
-        LogsFeature.appVersion = ""
+        CoreFeature.envName = ""
+        CoreFeature.packageVersion = ""
         fakeMessage = forge.anAlphabeticalString()
         testedLogger = Logger(mockLogHandler)
     }
 
     @AfterEach
     fun `tear down`() {
-        LogsFeature.envName = ""
-        LogsFeature.appVersion = ""
+        CoreFeature.envName = ""
+        CoreFeature.packageVersion = ""
```

Comment:
Same question as above.

Reply:
I prefer this to make sure the tests are clean in case there's a left over from another test suite.

Reply:
Doing so means we'll never know if another tests is leaking bad configuration. I'd rather have each test only setting up their own global dependencies and cleaning it. Otherwise, following your logic, you should also set  every single global object's state here just in case another test does something wrong.

Reply:
Yes my bad...they are actually not needed anymore after removing some code.